readme.md

```diff
@@ -268,6 +268,7 @@
 	- [Tutorials](https://github.com/ujjwalkarn/Machine-Learning-Tutorials#readme)
 	- [ML with Ruby](https://github.com/arbox/machine-learning-with-ruby#readme) - Learning, implementing, and applying Machine Learning using Ruby.
 	- [Core ML Models](https://github.com/likedan/Awesome-CoreML-Models#readme) - Models for Apple's machine learning framework.
+	- [H2O](https://github.com/h2oai/awesome-h2o) - Research, tutorials and applications that use the H2O distributed machine learning platform.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

The description should be about H2O itself, not the list.